components/ulp/project_include.cmake

```diff
@@ -86,15 +87,18 @@ function(__setup_ulp_project app_name project_path s_sources exp_dep_srcs)
 
         add_dependencies(${COMPONENT_LIB} ${app_name}_artifacts)
 
-        target_linker_script(${COMPONENT_LIB} INTERFACE ${CMAKE_CURRENT_BINARY_DIR}/${app_name}/${app_name}.ld)
         target_add_binary_data(${COMPONENT_LIB} ${CMAKE_CURRENT_BINARY_DIR}/${app_name}/${app_name}.bin BINARY)
     endif()
 endfunction()
 
 function(ulp_embed_binary app_name s_sources exp_dep_srcs)
-    __setup_ulp_project("${app_name}" "${idf_path}/components/ulp/cmake" "${s_sources}" "${exp_dep_srcs}")
+    __setup_ulp_project("${app_name}" "${idf_path}/components/ulp/cmake" "ulp_" "${s_sources}" "${exp_dep_srcs}")
+endfunction()
+
+function(ulp_embed_binary_prefix app_name prefix s_sources exp_dep_srcs)
```

Comment:
Nitpick, I think it would be nicer to add and optional PREFIX keyword argument to ulp_embed_binary, so that the usage would be 
`ulp_embed_binary(${ulp_app_name} "${ulp_sources}" "${ulp_exp_dep_srcs}" PREFIX "::ulp")`

Reply:
Ok, I need to learn how to do that and will do.